wgsl/index.bs

```diff
@@ -2008,32 +2008,42 @@ Issue: (dneto): Bitwise-complement is under discussion.  https://github.com/gpuw
     <tr><td>Precondition<td>Conclusion<td>Notes
   </thead>
   <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
+          *e2* : u32<br>
           *T* is *Int*
        <td class="nowrap">`e1 << e2` : *T*
-       <td>Shift *e1* left by *e2* bits (OpShiftLeftLogical)
-  <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
-          *T* is *IntVec*
-       <td class="nowrap">`e1 << e2` : *T*
+       <td>Shift left:<br>
+           Shift *e1* left, inserting zero bits at the least significant positions,
+           and discarding the most significant bits.
+           The number of bits to shift is the value of *e2* modulo the bit width of *e1*.
+           (OpShiftLeftLogical)
+  <tr><td>*e1* : vec*N*&lt;*T*&gt;<br>
+          *e2* : vec*N*&lt;u32&gt;<br>
+          *T* is *Int*
+       <td class="nowrap">`e1 << e2` : vec*N*&lt;*T*&gt;
        <td>Component-wise shift left (OpShiftLeftLogical)
   <tr><td>*e1* : u32<br>
           *e2* : u32<br>
        <td class="nowrap">`e1 >> e2` : u32
-       <td>Logical shift *e1* right by *e2* bits, i.e. inserting zero at most significant bits (OpShiftRightLogical)
-  <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
-          *T* is vec*N*&lt;u32&gt;
-       <td class="nowrap">`e1 >> e2` : *T*
+       <td>Logical shift right:<br>
+           Shift *e1* right, inserting zero bits at the most significant positions,
+           and discarding the least significant bits.
+           The number of bits to shift is the value of *e2* modulo the bit width of *e1*.
+           (OpShiftRightLogical)
+  <tr><td>*e1* : vec*N*&lt;u32&gt;<br>
+          *e2* : u32<br>
+       <td class="nowrap">`e1 >> e2` : vec*N*&lt;u32&gt;
        <td>Component-wise logical shift right (OpShiftRightLogical)
   <tr><td>*e1* : i32<br>
-          *e2* : i32<br>
+          *e2* : u32<br>
        <td class="nowrap">`e1 >> e2` : i32
-       <td>Arithmetic shift *e1* right by *e2* bits, i.e. replicating the sign bit of *e1* at most significant bits (OpShiftRightArithmetic)
-  <tr><td>*e1* : *T*<br>
-          *e2* : *T*<br>
-          *T* is vec*N*&lt;i32&gt;
-       <td class="nowrap">`e1 >> e2` : *T*
+       <td>Arithmetic shift right:<br>
```

Comment:
is there a value in calling it "arithmetic shift" versus something simpler, like a "signed shift"?

Reply:
"Arithmetic shift" is a common name for it.

https://en.wikipedia.org/wiki/Arithmetic_shift